roles/download/defaults/main.yml

```
@@ -1362,6 +1375,19 @@ downloads:
     groups:
     - k8s_cluster
 
+  cilium_hubblecli:
+    enabled: "{{ cilium_enable_hubble }}"
+    file: true
+    version: "{{ cilium_hubble_cli_version }}"
+    dest: "{{ local_release_dir }}/hubble"
+    sha256: "{{ cilium_hubblecli_binary_checksum }}"
+    url: "{{ cilium_hubblecli_download_url }}"
+    unarchive: true
+    owner: "root"
+    mode: "0755"
+    groups:
+    - k8s_cluster
```

@necatican has a good point as

> One thing to consider, though; should we install the cli on all the nodes or just the control plane nodes? It shouldn't make much difference; I'm just unsure about it.

kubectl is installed with the group kube_control_plane, so here can be changed to

```suggestion
    - kube_control_plane
```